src/components/SearchForm.js

```diff
@@ -0,0 +1,74 @@
+import { useContext, useState } from 'react';
+import { GoogleMapsContext } from '../context/GoogleMapsContext';
+import { MapCenterContext } from '../context/MapCenterContext';
+import planetSearch from '../assets/search-location-solid.svg';
+
+export const SearchForm = () => {
+  const [query, setQuery] = useState('');
+  const valueGoogleMapsAPI = useContext(GoogleMapsContext);
+  const valueMapCenter = useContext(MapCenterContext);
+  const { map, maps } = valueGoogleMapsAPI;
+  const { setNewCenterMap } = valueMapCenter;
+
+  const handleQuery = (e) => setQuery(e.target.value);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    var request = {
```

Comment:
Curious what the mentors have to say, but I've understood we should refrain from writing code that uses `var` since it is more likely to create possible errors later. Thoughts?

Reply:
Good eye, Jessica!  We had copied a couple of methods from the maps api page examples and didn't change the variable assignments to the modern best practices.